repos/system_upgrade/el7toel8/actors/migratebrltty/tests/component_test.py

```diff
@@ -0,0 +1,93 @@
+from leapp.snactor.fixture import current_actor_context
+from leapp.models import BrlttyMigrationDecision
+from leapp.reporting import Report
+import os
+
+BRLTTY_CONF='/etc/brltty.conf'
```

Comment:
I'm having bad feeling about running unit tests that modify files in /etc directory

Reply:
Mocked